keg_elements/forms/__init__.py

```diff
@@ -348,11 +348,50 @@ def wrapper(form):
 
 
 class Form(BaseForm):
+    """Base form with a bunch of QoL improvements
+
+    .. note:: Relying on the default field ordering can lead to unintuitive forms. It is possible to
+        override this by adding the ``__field_order`` class attribute. Set this class variable to a
+        tuple or list of field names (addressable via Form._fields['name_of_field']) and the form
+        will render in that order. You must include all the fields, except CSRF.  Forgetting a field
+        or adding one which doesn't exist will cause the form to raise a ``ValueError`` and the form
+        will not be rendered.
+
+    Example ::
+
+        class MyForm(Form):
+            __field_order = ('field1', 'field2',)
+
+            field1 = String('field1_label')  # Note that we don't use the label in the ordering
+            field2 = String()
+
+    """
     def __init__(self, *args, **kwargs):
         super(Form, self).__init__(*args, **kwargs)
         self._form_level_errors = []
         self.after_init(args, kwargs)
 
+    def __iter__(self):
+        # :rage: ... I have no idea why you need the class name, but alas, you do!
```

Comment:
You need the class name because you've marked this as a ["private"](https://docs.python.org/3/tutorial/classes.html?highlight=private#tut-private) field, by naming it with two leading underscores.

If you don't require the name mangling, it's probably easiest to just use a single leading underscore, or some other naming convention.

Reply:
TIL! Thanks